applications/ShapeOptimizationApplication/test_examples/06_bead_optimization_plate/analysis_parameters.json

```diff
@@ -0,0 +1,108 @@
+{
+  "problem_data"             : {
+      "problem_name"    : "Plate",
+      "parallel_type"   : "OpenMP",
+      "start_time"      : 0.0,
+      "end_time"        : 1.0,
+      "echo_level"      : 0
+  },
+  "solver_settings"          : {
+      "solver_type"     : "static",
+      "echo_level"      : 0,
+      "analysis_type"   : "linear",
+      "model_part_name" : "plate",
+      "domain_size"     : 3,
+      "time_stepping" : {
+          "time_step"       : 1.0
+      },
+      "model_import_settings"              : {
+          "input_type"     : "use_input_model_part",
+          "input_filename" : "plate"
+      },
+          "material_import_settings" :{
+          "materials_filename": "2D_material.json"
+      },
+      "line_search"                        : false,
+      "convergence_criterion"              : "residual_criterion",
+      "displacement_relative_tolerance"    : 0.0001,
+      "displacement_absolute_tolerance"    : 1e-9,
+      "residual_relative_tolerance"        : 0.0001,
+      "residual_absolute_tolerance"        : 1e-9,
+      "max_iteration"                      : 10,
+      "linear_solver_settings"             : {
+          "solver_type" : "eigen_pardiso_lu",
+          "scaling"     : false,
+          "verbosity"   : 0
+      },
+      "problem_domain_sub_model_part_list" : ["structure"],
+      "processes_sub_model_part_list"      : ["corner_points","load_point"],
+      "rotation_dofs"                      : true
+  },
+  "constraints_process_list" : [{
```

Comment:
the processes should be wrapped in a `processes` block
what you have now is deprecated and will eventually be removed
=> you GiD-interface seems to be quite old

Reply:
@philbucher Can you post here the way it should look according to the new interface? Thanks.

Reply:
where the deprecation-msg points to:https://github.com/KratosMultiphysics/Kratos/wiki/Common-Python-Interface-of-Applications-for-Users#analysisstage-usage